File: src/main/java/net/atos/client/zgw/ztc/model/ResultaattypeListParameters.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.client.zgw.ztc.model;

import java.net.URI;

import jakarta.ws.rs.QueryParam;

/**
 *
 */
public class ResultaattypeListParameters extends AbstractZTCListParameters {

  /**
   * URL-referentie naar het ZAAKTYPE van ZAAKen waarin resultaten van dit RESULTAATTYPE bereikt kunnen worden.
   */
  @QueryParam("zaaktype")
  private final URI zaaktype;

  public URI getZaaktype() {
    return zaaktype;
  }

  public ResultaattypeListParameters(final URI zaaktype) {
    this.zaaktype = zaaktype;
  }
}

File: src/main/kotlin/net/atos/client/zgw/ztc/model/ResultaattypeListParameters.kt
/*
 * SPDX-FileCopyrightText: 2021 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.client.zgw.ztc.model

import jakarta.ws.rs.QueryParam
import java.net.URI

/**
 *
 */
class ResultaattypeListParameters(
    /**
     * URL-referentie naar het ZAAKTYPE van ZAAKen waarin resultaten van dit RESULTAATTYPE bereikt kunnen worden.
     */
    @field:QueryParam("zaaktype") val zaaktype: URI
) : AbstractZTCListParameters()
